config/utils.py

```diff
@@ -0,0 +1,29 @@
+import json
+from dataclasses import dataclass
+
+
+@dataclass
+class VcapServices(object):
+
+    services: dict
+
+    @staticmethod
+    def from_env_json(json_string: str):
+        json_dict = dict(json.loads(json_string))
+        vcap_services = VcapServices(services=json_dict)
+        return vcap_services
+
+    def get_service_by_name(self, group_key: str, name: str) -> dict:
+        service_group = self.services.get(group_key)
+        if service_group:
+            for service in service_group:
+                if service.get("name") == name:
+                    return service
+            raise Exception("Service name '" + name + "' not found")
```

Comment:
could we use [formatted string literals](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) for these?

Reply:
Yep updated